src/RepoAutomation.Tests/BranchProtectionTests.cs

```diff
@@ -58,27 +58,52 @@ public async Task GetBranchProtectionPolicyThatDoesNotExistTest()
         }
     }
 
-    //[TestMethod]
-    //public async Task UpdateBranchProtectionPolicyTest()
-    //{
-    //    //Arrange
-    //    string owner = "samsmithnz";
-    //    string repoName = "RepoAutomation";
-    //    string branchName = "main";
-    //    string[] contexts = new string[]
-    //    {
-    //        "version",
-    //        "build (Linux_x64, linux-x64)",
-    //        "build (Windows_x64, windows-x64)"
-    //    };
+    [TestMethod]
+    public async Task UpdateBranchProtectionPolicyTest()
+    {
+        //Arrange
+        string owner = "samsmithnz";
+        string repoName = "RepoAutomation";
+        string branchName = "main";
+        //string[] contexts = new string[]
+        //{
+        //    "version"
+        //    //,
+        //    //"build (Linux_x64, linux-x64)",
+        //    //"build (Windows_x64, windows-x64)"
+        //};
 
-    //    //Act
-    //    bool result = await GitHubAPIAccess.UpdateBranchProtectionPolicy(base.GitHubId, base.GitHubSecret, owner, repoName,
-    //        branchName, contexts);
+        RequiredStatusCheckPut requiredStatusCheck = new()
```

Comment:
This is almost the same as null - should check the required status checks checkbox with no options. Doesn't work, returns 422